brands/shop/car_repair.json

```
@@ -214,6 +214,17 @@
       "shop": "car_repair"
     }
   },
+  "shop/car_repair|Monro Muffler Brake": {
+    "nocount": true,
+    "countryCodes": ["us"],
```

Maybe this should also have a `match` property for `shop/car_repair|Monro`?

```suggestion
    "countryCodes": ["us"],
    "match": ["shop/car_repair|Monro"],
```